lib/ansible/runner.rb

```diff
@@ -196,6 +196,8 @@ def run_via_cli(hosts, credentials, env_vars, extra_vars, tags: nil, ansible_run
         validate_params!(env_vars, extra_vars, tags, ansible_runner_method, playbook_or_role_args)
 
         base_dir = Dir.mktmpdir("ansible-runner")
+        debug    = true if verbosity.to_i >= 5
+        debug  ||= true if env_vars["ANSIBLE_KEEP_REMOTE_FILES"] == "1"
```

Comment:
```suggestion
        debug = (verbosity.to_i >= 5) || (env_vars["ANSIBLE_KEEP_REMOTE_FILES"] == "1")
```

Reply:
Weird - I had a comment here yesterday and it never sent :(

---

2 comments:

1. the `||=` is not doing anything here.  If `debug` is false or nil, then it will be set to true, and if it's true, it's already true.  Might as well just use `=`.
2. Unless the env var is something that is defined by ansible, prefer `if env_vars["ANSIBLE_KEEP_REMOTE_FILES"]` over `if env_vars["ANSIBLE_KEEP_REMOTE_FILES"] == "1"`, because it's not uncommon for that env var to be set to `true` or some other value.  Elsewhere in our code we just check for the presence of the ENV var as opposed to a specific value